laravel/app/Http/Controllers/AdminController.php

```diff
@@ -539,18 +539,18 @@ public function postEditDB()
             if (strlen($_POST['assignment']) > 0 && Assignment::where('schedule', $id)->exists())
             {
                 $assignment = Resident::where('email', $_POST['assignment'])->value('id');
-                Assignment::where('schedule', $id)->update([
-                    'resident'=>$assignment
+                Assignment::where('schedule_id', $id)->update([
+                    'resident_id'=>$assignment
                 ]);
             }
             elseif (strlen($_POST['assignment']) > 0)
             {
                 $assignment = Resident::where('email', $_POST['assignment'])->value('id');
                 $date = $_POST['date'];
                 Assignment::insert([
-                    'date'=>$date, 'resident'=>$assignment,
-                    'attending'=>$_POST['lead_surgeon_code'],
-                    'schedule'=>$id,
+                    'date'=>$date, 'resident_id'=>$assignment,
+                    'attending_id'=>$_POST['lead_surgeon_code'],
```

Comment:
Isn't the attending col removed from assignment?

Reply:
Yep, should be fixed on next push!